Makefile

```diff
@@ -26,31 +26,29 @@ install:
 
 dist:
 	@mkdir -p  $(build)/$(NAME)-$(VERSION)/
-	rsync -HaS --exclude ".svn" --exclude "$(build)" * $(build)/$(NAME)-$(VERSION)/
+	rsync -aHS --exclude ".git" --exclude "$(build)" * $(build)/$(NAME)-$(VERSION)/
 	cd $(build); tar --gzip -cf $(NAME)-$(VERSION).src.tgz $(NAME)-$(VERSION)/; cd -
 
 sources: dist
 	cp $(build)/$(NAME)-$(VERSION).src.tgz .
 
-deb: dist
```

Comment:
Are we really just dropping debs?

Reply:
It's missing all the debian directory, so no way to create a clean debian pacakges like this.
Best to add later the required stuff to have a clean package.